Updates.list

```diff
@@ -1,4 +1,5 @@
  971033 9 Update for Windows Activation Technologies
+2882822 9 Update adds ITraceRelogger interface support.

```

Comment:
Yes, and? According to the update description, removing it may botch the IE page debugger.